### ISSUE TITLE
v2 preview overwrite settings to default

### ISSUE THREAD

**Issue by Commissar0617** [HaveAGitGat/Tdarr#314]
windows version... can't get server or node to run because handbrake always fails because it resets the config jsons every time i  try to run it.

**Comment by HaveAGitGat** [comment]
Seems like you have an invalid config JSON. The config needs to be valid as several different things use it so it gets overwritten with defaults if it's not valid.

I'm assuming there's a incorrect double/single quote in the handbrake path or something like that. Most text editors tell you if there's invalid json but you can also use something like the following to check the JSON is valid:
https://jsonlint.com/

**Comment by Commissar0617** [comment]
![image](https://user-images.githubusercontent.com/5993529/104858090-709b2100-58e2-11eb-9229-3b85afc1d5e8.png)
this is what im trying to set my json to

**Comment by Commissar0617** [comment]
Error: Parse error on line 7:
...,	"handbrakePath": "C:\Program Files\Ha
----------------------^
Expecting 'STRING', 'NUMBER', 'NULL', 'TRUE', 'FALSE', '{', '[', got 'undefined'

**Comment by wsegatto** [comment]
Try to run with a clean JSON in the Config folder and only change what shows as an error.
e.g. If I manually indicate the path to ffmpeg and handbrake, handbrake gives an error. If I run it without paths, it says ffmpeg has an error, therefore I left handbrake blank and ffmpeg to the path /usr/bin/ffmpeg.

**Comment by Commissar0617** [comment]
Yep. It was just failing because handbrake is apparently not a valid command. Ill be getting it set up on a linux vm sometime this week.

**Comment by HaveAGitGat** [comment]
1.99.13 logs  the error in the terminal so you can see what the issue is. It also creates a backup of the config before overwriting it. Let me know if you still get the error.

**Comment by wsegatto** [comment]
Only on Windows for now, I guess?
I'm having strange "copying" issues when the output folder is at a remote location. But I see no relevant logs even with verbose logging. The "copying" queue gets stuck with no "retry" button.

**Comment by HaveAGitGat** [comment]
`Only on Windows for now, I guess?`

For Windows, macOS and Linux. The latest package (.14), is also available on Arm.

**Comment by HaveAGitGat** [comment]
The error will be logged in the terminal if there's an issue with the config:

` Error reading config file:SyntaxError: Unexpected token } in JSON at position 97`

Try 2.00.01 using this method: https://github.com/HaveAGitGat/Tdarr/issues/327#issuecomment-767516511

Please re-open if there's still an issue.
